config/secrets.yml

```diff
@@ -14,7 +14,7 @@ development:
   secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
 
 test:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: 7053b205c9559f83960f18d23f10d36cacd737c0e17e0dd71cbb90d2f3e6
```

Comment:
@rhcarvalho which also makes me thing that we can use template :-) just do substitution in `script` field in postCommit hook:

```
script: SECRET_BASE_BLA=$SECRET_BASE_BLA bundle exec rake test
```